website/source/docs/agent/checks.html.md

```diff
@@ -50,14 +50,15 @@ There are several different kinds of checks:
   to check a simple HTTP operation. By default, HTTP checks are `GET` requests
   unless the `method` field specifies a different method. Additional header
   fields can be set through the `header` field which is a map of lists of
-  strings, e.g. `{"x-foo": ["bar", "baz"]}`. By default, HTTP checks will be
-  configured with a request timeout equal to the check interval, with a max of
-  10 seconds. It is possible to configure a custom HTTP check timeout value by
-  specifying the `timeout` field in the check definition. The output of the
-  check is limited to roughly 4KB. Responses larger than this will be truncated.
-  HTTP checks also support TLS. By default, a valid TLS certificate is expected.
-  Certificate verification can be turned off by setting the `tls_skip_verify`
-  field to `true` in the check definition.
+  strings, e.g. `{"x-foo": ["bar", "baz"]}`. By default, HTTP checks are set to
```

Comment:
This reads a little confusingly, I think you mean the timeout but it sounds more like the Interval.

```suggestion
  strings, e.g. `{"x-foo": ["bar", "baz"]}`. By default, HTTP checks will be configured to timeout after
```